### PR TITLE
Adds slime people

### PR DESCRIPTION
**Pull request by Zumorica**
Adds a slime person mob.
They are *exactly* the same as humans currently, and only serve as an example for how to add different species.
Skin tone is hardcoded to be the same as humans because our code sucks.

**Changelog**
:cl:
- add: Added a basic slime person mob.

### CODE REVIEW

File: Resources/Prototypes/Entities/Constructible/Walls/walls.yml [hunk 1]
```diff
@@ -554,3 +554,10 @@
   - type: IconSmooth
     key: walls
     base: wood
+  - type: Flammable
+  - type: AtmosExposed
+  - type: Appearance
+    visuals:
+      - type: FireVisualizer
+        sprite: Mobs/Effects/onfire.rsi
+        normalState: Monkey_burning
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Constructible/Walls/walls.yml:563]: are these slime ppl changes
image

Comment (PaulRitter) [reply Resources/Prototypes/Entities/Constructible/Walls/walls.yml:563]: also wat. `monkey_burning`
whats happening here

**Review by PaulRitter — CHANGES_REQUESTED** [review]

**Comment by Peptide90** [comment]
Would there be much point porting other species or should I wait until more systems are in rather than having to maintain more stuff as changes happen?

**Comment by SweptWasTaken** [comment]
> 
> 
> Would there be much point porting other species or should I wait until more systems are in rather than having to maintain more stuff as changes happen?

Nothing player controllable.

File: Resources/Prototypes/Entities/Constructible/Walls/walls.yml (hunk 1 above)
Comment (Zumorica) [reply Resources/Prototypes/Entities/Constructible/Walls/walls.yml:563]: oops, this is a lil' test I was doing for a different thing! I'll fix it..... tomorrow

**Comment by Zumorica** [comment]
> Would there be much point porting other species or should I wait until more systems are in rather than having to maintain more stuff as changes happen?

As he said, there's really not much point to it right now. This one was done on a whim to see how easy it'd be to do, to be honest.

**Comment by PJB3005** [comment]
Vox aren't the first species...

**Comment by SweptWasTaken** [comment]
> 
> 
> Vox aren't the first species...

eat sand furry

**Review by PaulRitter — APPROVED** [review]

**Pull request merged**